discovery-provider/src/tasks/playlists.py

```diff
@@ -143,7 +148,7 @@ def parse_playlist_event(
 
     if event_type == playlist_event_types_lookup["playlist_track_added"]:
         if getattr(playlist_record, 'playlist_contents') is not None:
-            logger.warning('playlist event playlist_track_added')
+            logger.info('playlist event playlist_track_added')
```

Comment:
def a good move to move these to `info` level logs - can we add a little more identifying information to this string so it might be more useful?